governance/charter.md

```diff
@@ -0,0 +1,102 @@
+# SIG-Security Charter (draft)
+
+This charter describes operations as a [CNCF SIG](https://github.com/cncf/toc/blob/master/sigs/). The [Focus](#focus) section below describes what is in and out of scope,
+and [Governance](#governance) section describes how our operations are consistent with CNCF policies with links to more detailed documents.
+
+**Mission:** to reduce risk that cloud native
+applications expose end user data or allow other unintended access.
+
+## Motivation
+Security has been an area in which open source can flourish and sometimes has done so; however, with cloud native platforms and applications, security has received less attention than other areas of the cloud native landscape.
+
+This means that there is less visibility about the internals of security projects, and fewer projects being deeply integrated into cloud native tooling. While there are many open source security projects, there are fewer security experts focused on the cloud native ecosystem. This has contributed to a culture where people feel they cannot understand how to securely set up and operate cloud native systems, due to obscurity and uncertainty. Yet the cloud native principles, although they mean change, have encouraged the development of tools that help manage fast changing environments, and which have the promise of both simplifying and improving security.
```

Comment:
s/Yet the cloud native principles, although they mean change,/Cloud native principles

Reply:
nice simplification